accounts/views.py

```diff
@@ -32,4 +32,21 @@ def post(self, request):
             )
             return JsonResponse ({"MESSAGE":"SUCCESS"}, status = 201)
         except KeyError:
-            return JsonResponse ({"MESSAGE":"KEY_ERROR"}, status = 400)
\ No newline at end of file
+            return JsonResponse ({"MESSAGE":"KEY_ERROR"}, status = 400)
+
+class SigninView(View):
+    def get(self, request):
+        data = json.loads(request.body)
+        name = Account.objects.get(email=data["email"])
```

Comment:
1. keyerror가 발생 할 수 있는 상황입니다.
2. email이 db에 없을 경우에 대한 예외 처리 추가해 주세요.
```suggestion
        user = Account.objects.get(email=data["email"])
```

Reply:
넵 try문 안으로 넣어 KeyError 상황을 처리하고 email이 db에 없는 경우도 조건을 넣어 수정했습니다